vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go

```diff
@@ -370,7 +373,10 @@ func writeCon(fpath string, val string) error {
 	} else {
 		_, err = out.Write(nil)
 	}
-	return err
+	if err != nil {
+		return errors.Wrapf(err, "failed to set %s on procfs", fpath)
+	}
+	return nil
```

Comment:
Just notice this; the conditional (`if ..`) actually isn't even needed, because `errors.Wrap(f)` will return `nil` if the error passed is `nil`